CONTRIBUTING.rst

```diff
@@ -0,0 +1,263 @@
+
+Commit Sequences
+~~~~~~~~~~~~~~~~
+
+Update your branch to be baseline on the latest master if new
+developments were merged while you were developing. Please prefer
+**rebasing** to merging, as explained in `this
+tutorial <https://coderwall.com/p/7aymfa/please-oh-please-use-git-pull-rebase>`__.
+Note that if you do rebases after review have started, they will be
+cancelled, so at this point it may be more appropriate to do a pull.
+
+Clean-up your commit sequence. If your are not familiar with git, `this
+good tutorial <https://www.atlassian.com/git/tutorials/rewriting-history>`__
+on the subject may help you.
+
+Focus on committing 1 logical change at a time. See `this article
+<https://github.com/erlang/otp/wiki/writing-good-commit-messages>`__
+on the subject.
+
+
+Submitting a Pull Request
+-------------------------
+
+PR Title
+~~~~~~~~
+
+For each new release, so please follow the following rules:
+
+-  Provide a concise and self-descriptive title (see 'Issue Title').
+-  Do not include the applicable issue number in the title, do it in the PR body (see 'PR Body' section).
+-  If the PR is not ready for review, convert it to a draft.
+
+PR Body
+~~~~~~~
+
+Describe what the PR is about, explain the approach and possible
+drawbacks. Don't hesitate to repeat some of the text from the related
+issue (easier to read than having to click on the link).
+
+If the PR fixes issue(s), indicate it after your introduction:
+``Fixes #XXXX, Fixes #YYYY``. Note: it is important to respect the
+syntax above so that the issue(s) will be closed upon merging the PR.
+
+PR Labels
+~~~~~~~~~
+
+Add the labels that best describe your PR ('enhancement', 'documentation', etc...
+
+
+Work in progress 
+~~~~~~~~~~~~~~~~
+
+If your PR is not ready for review yet, you can convert it to a "Draft", so the team is informed.
+
+A draft pull request is styled differently to clearly indicate that it’s in a draft state. 
+Merging is blocked in draft pull requests. Change the status to “Ready for review” near the 
+bottom of your pull request to remove the draft state and allow merging according to your 
+project’s settings. 
+
+Continuous Integration
+~~~~~~~~~~~~~~~~~~~~~~
+
+The PR can't be merged if `Github Actions "Run
+tests" <https://travis-ci.com/github/shimming-toolbox/shimming-toolbox-py>`__
+hasn't succeeded. If you are familiar with it, consult the test results
+to fix the problem.
+
+Reviewers
+~~~~~~~~~
+
+Any changes submitted for inclusion to the master branch will have to go
+through a
+`review <https://help.github.com/articles/about-pull-request-reviews/>`__.
+
+Only request a review when you deem the PR as "good to go". If the PR is
+not ready for review, convert it to a "Draft".
+
+Github may suggest you to add particular reviewers to your PR. If that's
+the case and you don't know better, add all of these suggestions. The
+reviewers will be notified when you add them.
+
+Versioning
+----------
+--> TODO
```

Comment:
why not using [that](https://github.com/neuropoly/spinalcordtoolbox/blob/master/CONTRIBUTING.rst#versioning)?

Reply:
That's what I was thinking but I wanted to know if it was ok for you.